armory/baseline_models/pytorch/ucf101_mars.py

```diff
@@ -54,13 +49,13 @@ def make_model(model_status="ucf101_trained", weights_file=None):
         opt.batch_size = 32
         opt.ft_begin_index = 4
 
-        opt.pretrain_path = filepath
+        opt.pretrain_path = weights_path
```

Comment:
This is a minor change in behavior — previously, if `weights_file` was `None`, this would be an error, as `filepath` would be undefined; now if `weights_path` is `None` it will also be `None`